front-end/src/App.js

```diff
@@ -13,6 +14,7 @@ class App extends Component {
           <Route exact path="/" component={HomePage} />
           <Route exact path="/stripe" component={Stripe}/>
           <Route exact path="/navbar" component={Navbar}/>
+          <Route exact path="/register" component={RegistrationForm}/>
```

Comment:
we don't need a route to registration form.  It exists in the homepage component as a pop up modal